src/blockchain/utils.ts

```diff
@@ -396,6 +409,8 @@ export const getXAssetCLRTokenSymbol = (symbol: IXAssetCLR): ICLRToken => {
   switch (symbol) {
     case AAVE_X_AAVE_A_CLR:
       return { 0: AAVE, 1: X_AAVE_A }
+    case ALPHA_X_ALPHA_A_CLR:
+      return { 0: AAVE, 1: X_AAVE_A }
```

Comment:
Should return `ALPHA` & `X_ALPHA_A`. Please confirm the order of asset once xAssetCLR pool is deployed.